concepts/observability-strategy/configurable-tracing/pocs/instrumentation/README.md

```diff
@@ -0,0 +1,136 @@
+
+# Testing
+
+Test was performed on Kyma version 2.6 deployed on Kubernetes version 1.23.9 using the [Synthetic Load Generator utility](https://github.com/Omnition/synthetic-load-generator) running for a minumum of one hour.
+Test results are reproducible by using the parameters described in this document. It is important to know that this utility has a few configurable parameters, which can impact the result of the tests.
+The parameters used in this test are:
+
+- flushIntervalMillis: 1000
+- MaxQueueSize: 100
+- Submission rate: 100000 span/sec
+
+OpenTelemetry Collector:
+- 1 CPU
+- 2 GiB Memory
+- Processor memory limiter: 1000 Mib with a spike limit 256 Mib, and an interval check of one second.
+
+Test goal is, find a maximum sustained rate with given default configuration and allocated resources (CPU and Memory).
+
+Test setup in total reach 3248176 (900 traces/sec.) traces with total 162408800 (45K span/sec.) spans.
+
+| ![CPU Utilization](assets/cpu.jpg) |
+| :--: |
+| Fig. 1 CPU utilization |
+
+
+| ![MemoryUtilization](assets/memory.jpg) |
+| :--: |
+| Fig. 2 Memory utilization |
+
+| ![Receive Bandwidth](assets/receive_bandwidth.jpg) |
+| :--: |
+| Fig. 3 Receive Bandwidth |
+
+| ![Transmit Bandwidth](assets/transmit_bandwidth.jpg) |
+| :--: |
+| Fig. 4 Transmit Bandwidth |
+
+| ![Rate of Received Packets](assets/receive_rate.jpg) |
+| :--: |
+| Fig. 5 Rate of Received Packets |
+
+| ![Rate of Transmitted Packets](assets/transmit_rate.jpg) |
+| :--: |
+| Fig. 6 Rate of Transmitted Packets |
+
+With using head based sampling, when higher rates needed, either:
+
+- Divide traffic to different collector
+- Scale-up by adding more CPU/Memory resources (doubling CPU and Memory will result two time more rate).
+- Scale-out by adding one or more collector behind a load balancer or a Kubernetes service.
+
+With tail based sampling either:
+- Scale-up by adding more CPU/Memory resources (doubling CPU and Memory will result two time more rate).
+- Scale-out by adding one or more collector behind a load balancer or a Kubernetes service, but the load balancer have to support **traceID-based** routing because all span of a given traceID need to be received by the dame collector instance.
+
+## Queue Test
+
+The OpenTelemetry pipelines offers a set of queue and retry mechanism, in case of backend outages. 
+The queue mechanism consist on two stages, first one is on the processor level in this case **Batch Processor**, 
+the batch processor queue is pretty simple and will queue **spans** received from any receiver before processed. This processor support both size and time based batching.
+The batch processor queue is an only in-memory queue and offers the following configuration possibilities:
+
+- **send_batch_size**: Number of spans, which a batch will be sent regardless of the timeout.
```

Comment:
Not sure if I understood this correctly. Please make sure this is correct. 

```suggestion
- **send_batch_size**: Number of spans that a batch receives, regardless of the timeout.
```

Reply:
Maybe something like "Number of spans that a single batch can contain, regardless of the timeout."

Reply:
Nice, that sounds much better!